RBDM/Base.h

```diff
@@ -0,0 +1,14 @@
+#ifndef BaseTypes_h_
+#define BaseTypes_h_
+
+#include<iostream>
+
+class Base
```

Comment:
Bad name for class. I can't even imagine what is this class for, because name 'Base' don't tell nothing.